openshift-with-appstudio-test/e2e/const.go

```diff
@@ -0,0 +1,12 @@
+package e2e
+
+const (
+	testNamespace          = "jvm-build-service-test-namespace-"
+	maxNameLength          = 63
+	randomLength           = 5
+	maxGeneratedNameLength = maxNameLength - randomLength
+
+	testProjectGitUrl = "https://github.com/stuartwdouglas/hacbs-test-project"
+	// TODO update this
+	testProjectDevfileUrl = "https://raw.githubusercontent.com/psturc/shaded-java-app/main/devfile.yaml"
```

Comment:
@gabemontero @stuartwdouglas we need to point an AppStudio Component to a devfile (that is then parsed by build-service which then decides which pipeline is going to be used for building the component). I used a devfile from my other repo, which I guess is fine now, but I was wondering if we could make it a part of that `hacbs-test-project` repo, so it lives together with the source code - wdyt?

Reply:
@stuartwdouglas owns the test repos, so I'll defer to him (though I am betting it is OK to add a devfile)

Reply:
created a PR for that: https://github.com/stuartwdouglas/hacbs-test-project/pull/1